src/System.Windows.Forms/src/System/Windows/Forms/RelatedImageListAttribute.cs

```diff
@@ -13,19 +11,19 @@ namespace System.Windows.Forms
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
     public sealed class RelatedImageListAttribute : Attribute
     {
-        private readonly string relatedImageList;
+        private readonly string _relatedImageList;
 
         public RelatedImageListAttribute(string relatedImageList)
```

Comment:
We don't make any assumptions or assertions about `relatedImageList`
```suggestion
        private readonly string? _relatedImageList;

        public RelatedImageListAttribute(string? relatedImageList)
```

ditto below

Reply:
This is currently used only with strings such as `[RelatedImageList("ListView.GroupImageList")]` in the codebase and the property is called like `RelatedImageList.Split('.');` in a couple of places. So, since there is no null check before `Split` I thought it is better to make it non-nullable as a suggestion.

Reply:
This is used by (at least) the Windows Forms designer, and it's checked for nulls.
This is a public API which doesn't performs any null checks, so it means it can be a null :)